### ISSUE TITLE
Periods in a Path Template cause error

### ISSUE THREAD

**Issue by pquerna** [daveshanley/vacuum#282]
I have a path template like this:
```
/shelves/{shelf}/books/{book.id}
```

this causes vacuum to report an error that the path param is unused in the path:
```
# Operations Issues

Line / Column | Severity | Message                                                                     | Path
(198:5)       | error    | parameter `book.id` must be used in path `/shelves/{shelf}/books/{book.id}` | $.paths.['/shelves/{shelf}/books/{book.id}']
```

I *think* the issue is this regexp, which is used to pull out template keys, and isn't matching a string with a period in it:

https://github.com/daveshanley/vacuum/blob/main/functions/openapi/path_parameters.go#L48

**Comment by daveshanley** [comment]
Thanks for finding this. A fix will be available in the next release.

**Comment by daveshanley** [comment]
This has been fixed in `v0.1.6`.
